build_n_deploy/naiserator/naiserator_prod.yaml

```diff
@@ -25,8 +25,9 @@ spec:
   istio:
     enabled: false
   ingresses:
-    - https://familie-ef-sak-frontend.nais.adeo.no
-    - https://alenemedbarn.nais.adeo.no
+    - https://familie-ef-sak-frontend.prod-fss.nais.io
+    - https://alenemedbarn.prod-fss.nais.io
+    - https://ensligmorellerfar.prod-fss.nais.io
```

Comment:
Burde vel kunne fjerne de 2 første her, ettersom konfigurerApp.ts ikke virker for de likevel?